backend/src/akvo/flow_services/geoshape_export.clj

```diff
@@ -22,9 +22,9 @@
             [akvo.flow-services.util :as util]
             [clojure.string :as s])
   (:import [java.util UUID TimeZone]
-           (org.waterforpeople.mapping.dataexport.service BulkDataServiceClient)
-           (org.waterforpeople.mapping.dataexport ExportImportUtils)
-           (java.text SimpleDateFormat)))
+           [org.waterforpeople.mapping.dataexport.service BulkDataServiceClient]
```

Comment:
After checking other code, it seems that IntelliJ/Cursive is the one adding `(:import )` as list when you use automatic import feature.  See: _Class Completion_ - https://cursive-ide.com/userguide/editing.html

Reply:
Yes I know, but is that a good enough excuse to not doing it? Personally, I think it is not important the [ vs ( because I am more used to (, but that is me and my personal background. 

Happy to do it either way but this wont be the last time that you need to remind me about the [